src/util/options.cpp

```diff
@@ -90,14 +92,12 @@ const optionst::value_listt &optionst::get_list_option(
 /// Returns the options as JSON key value pairs
```

Comment:
Worth noting this is now lazy, so calling this then mutating `optionst` then trying to iterate over the range may be undefined behaviour

Reply:
Please note that the return type of the function is still a value of type `json_objectt`, not a range. So there is no laziness beyond the scope of this function.

Reply:
@thomasspriggs might be worth adding an explicit constructor call to make that more obvious?